### PR TITLE
fix: bump dependencies to avoid vulnerability alerts

### PR DESCRIPTION
**Pull request by padamstx**
This PR contains a few changes to our dependency versions to avoid vulnerability alerts.
I also removed the contents of the .cveignore file since that is no longer needed.

##### Checklist

- [x] `npm test` passes (tip: `npm run lint-fix` can correct most style issues)
- [x] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: .cra/.cveignore [hunk 1]
```diff
@@ -1,6 +1,2 @@
 [
-    {
```
Comment (padamstx) [inline .cra/.cveignore:2]: An adhoc scan build shows that this exclusion is no longer needed.

File: package.json [hunk 2]
```diff
@@ -36,11 +36,10 @@
     "url": "https://github.com/IBM/node-sdk-core/issues"
   },
   "devDependencies": {
-    "@microsoft/api-documenter": "^7.13.68",
-    "@microsoft/api-extractor": "^7.18.19",
+    "@microsoft/api-documenter": "^7.13.76",
+    "@microsoft/api-extractor": "^7.19.2",
     "@types/debug": "^4.1.7",
     "@types/lodash.isempty": "^4.4.6",
-    "@types/dotenv": "^8.2.0",
```
Comment (padamstx) [inline package.json:43]: Messages displayed by npm when running "npm install" indicate that this dependency is simply not needed.

**Review by dpopp07 — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**